cyral/provider.go

```diff
@@ -170,6 +170,7 @@ func Provider() *schema.Provider {
 			"cyral_repository_datamap":          resourceRepositoryDatamap(),
 			"cyral_repository_identity_map":     resourceRepositoryIdentityMap(""),
 			"cyral_repository_local_account":    resourceRepositoryLocalAccount(),
+			"cyral_repository_user_account":     resourceRepositoryUserAccount(),
```

Comment:
I think we should call this `database_account` because that's the terminology we're using in Jeeves.